modules/tinymce/src/themes/silver/main/ts/ui/core/complex/StyleFormat.ts

```diff
@@ -30,7 +30,7 @@ export const defaultStyleFormats: AllowedFormat[] = [
       { title: 'Strikethrough', icon: 'strike-through', format: 'strikethrough' },
       { title: 'Superscript', icon: 'superscript', format: 'superscript' },
       { title: 'Subscript', icon: 'subscript', format: 'subscript' },
-      { title: 'Code', icon: 'code', format: 'code' }
+      { title: 'Code', format: 'code' }
```

Comment:
Okay, this is incorrect after looking more. The `Format` menu has this as using `sourcecode` so removing the icon is incorrect and it should instead be fixed from `code` -> `sourcecode`.

However that brings up a much larger issue that needs design input. Should style formats even be using icons? It did in v4, however I'm not convinced it makes sense as it makes the style formats menu rather inconsistent and the formats are rendered in place here.

Marking as "requesting changes" until we get feedback about what should happen with these.

- [ ] Get design feedback/approval for this
- [ ] Add back icon if design wants to show icons for style formats

Reply:
@lostkeys what do you think here? To clarify it's the `styleselect` part of the toolbar/menu I'm referring to:
![image](https://user-images.githubusercontent.com/1575550/81524389-59d81e00-9394-11ea-9a1f-a46e317e3581.png)

Reply:
I'd say _no_ to icons here. These options are user configurable and currently there is no option to provide icons to custom formats AFAIK. I might choose to call Italic "Emphasize" instead, then there would be a weird blank space in there. We would also inevitable get the question on how to define icons for custom formats and replying "no you can't" is kind of meh.

Reply:
All good and valid points, thanks @lostkeys! :+1:

Reply:
This should be fixed now.